pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -923,11 +927,28 @@ type AuditPolicy struct {
 
 // StructuredAuthentication contains authentication config for kube-apiserver.
 type StructuredAuthentication struct {
-	// ConfigMapName is the name of the ConfigMap in the project namespace
-	// which contains AuthenticationConfiguration for the kube-apiserver.
+	// ConfigMapName is the name of the ConfigMap in the project namespace which contains AuthenticationConfiguration
+	// for the kube-apiserver.
 	ConfigMapName string `json:"configMapName" protobuf:"bytes,1,opt,name=configMapName"`
 }
 
+// StructuredAuthorization contains authorization config for kube-apiserver.
+type StructuredAuthorization struct {
+	// ConfigMapName is the name of the ConfigMap in the project namespace which contains AuthorizationConfiguration for
+	// the kube-apiserver.
+	ConfigMapName string `json:"configMapName" protobuf:"bytes,1,opt,name=configMapName"`
+	// Kubeconfigs is a list of references for kubeconfigs for the authorization webhooks.
+	Kubeconfigs []AuthorizerKubeconfigReference `json:"kubeconfigs" protobuf:"bytes,2,rep,name=kubeconfigs"`
```

Comment:
Don't we need to adjust the [reference controller](https://github.com/gardener/gardener/tree/master/pkg/controllermanager/controller/shoot/reference) to consider those fields?